x/ccv/provider/keeper/relay.go

```diff
@@ -122,19 +126,8 @@ func (k Keeper) SendValidatorUpdates(ctx sdk.Context) {
 	k.IterateConsumerChains(ctx, func(ctx sdk.Context, chainID string) (stop bool) {
 		// check whether there is an established CCV channel to this consumer chain
 		if channelID, found := k.GetChainToChannel(ctx, chainID); found {
-			// send all the pending ValidatorSetChangePackets to the consumer chain
```

Comment:
Necessary refactor to reduce cognitive complexity of this method after adding error checks